recipes/fenics-oasis/meta.yaml

```diff
@@ -7,14 +7,15 @@ package:
 
 source:
   url: https://github.com/mikaem/Oasis/archive/{{ version }}.tar.gz
-  sha256: 4456d5f042292e30bffda0487b57e282fe6e776cb6cdc832140072983b2c9d8d
+  sha256: a610ddd83dcd514a193ed3dc7933475995ee850582e64e4aec90b27d28241592
 
 build:
   entry_points:
     - oasis=oasis.run_oasis:main
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
+  skip: true  # [win]
```

Comment:
It is OK for noarch package to fails on Windows with a missing dependency. We prefer noarch b/c it is faster to build and the package will be ready when the missing dependency is available on Windows.